www/source/partials/docs/_using-hab-continuous-deployment.html.md.erb

```diff
@@ -0,0 +1,56 @@
+
+If you have already uploaded your package to a channel and wish to promote it to a different channel, use the `hab pkg promote` subcommand as shown below.
+
+```bash
+$ hab pkg promote -z <OAuth_token> <origin>/<package>/<version>/<release> stable
+```
+
+## Combining an Update Strategy with Channels
+
+By using both channels and either the `at-once` or `rolling` update strategies, you can automatically update packages in a given channel as shown below:
+
+![Promoting packages through channels](/images/infographics/habitat-promote-packages-through-channels.png)
+
+Using the infographic a guide, a typical continuous deployment scenario would be as follows:
+
+1. You build a new version of your package either through Builder or a local Studio environment.
+2. When you are ready to roll out a new version of the package, you promote that package to the channel corresponding to the intended environment (e.g. `dev`). You can have multiple service groups in the same environment pointing to different channels, or the same channel.
+3. An existing set of running Supervisors in that service group see that the channel they are subscribed to has an update, so they update their underlying Habitat package, coordinating with one another per their update strategy, and restart their service.
+4. When you are ready, you then promote that version of your package to a new channel (e.g. `acceptance`). The running Supervisors in that group see the update and perform the same service update as in step 3. You repeat steps 3 and 4 until your package makes its way through your continuous deployment pipeline.
+
+Configuring a Supervisor's update strategy to point to a channel ensures that new versions of the application do not get deployed until the channel is updated, thereby preventing unstable versions from reaching environments for which they are not intended.
+
+To start a Supervisor with an update strategy and pointing to a channel, specify them as options when starting a service.
+
+    $ hab start <origin>/<package> --strategy rolling --channel test
+
+While that service is running, update your package, rebuild it, and then promote it to the same channel that the previous release of that serviceis currently running in (e.g. `test`). Those running instances should now update according to their update strategy.
+
+## Demoting a Package from a Channel
+
+If you need to unassociate a channel from a specific package release, you can do so using the `hab pkg demote` subcommand. Packages can be demoted from all channels except `unstable`.
+
+```bash
+$ hab pkg demote -z <OAuth_token> <origin>/<package>/<version>/<release> test
+```
+
+The Builder UI for that package release and `hab pkg channels` will both reflect the removal of that channel.
```

Comment:
Since we've been talking about supervisors automatically recognizing and acting on changes to package channels, my immediate question is whether, if I demote a package from a channel, the supervisor will pull down the most recent package in the channel for which it was started. I don't *think* it works this way, but I'm not sure, and I suspect others reading this will have the same question.